tests/network-tests/src/fixtures/council/ElectCouncilFixture.ts

```diff
@@ -35,41 +37,88 @@ export class ElectCouncilFixture extends BaseQueryNodeFixture {
     )
     await new FixtureRunner(addStakingAccountsFixture).run()
 
-    const votersStakingAccounts = (await this.api.createKeyPairs(numberOfVoters)).map(({ key }) => key.address)
-    await api.treasuryTransferBalanceToAccounts(votersStakingAccounts, voteStake)
-
     // Announcing stage
     await this.api.untilCouncilStage('Announcing')
 
-    const applyForCouncilTxs = candidatesMemberIds.map((memberId, i) =>
-      api.tx.council.announceCandidacy(
-        memberId,
-        candidatesStakingAccounts[i],
-        candidatesMemberAccounts[i],
-        councilCandidateStake
+    const announceCandidacyFixture = new AnnounceCandidacyFixture(
+      this.api,
+      this.query,
+      new Map(
+        candidatesMemberAccounts.map((account, i) => {
+          return [
+            account,
+            {
+              memberId: candidatesMemberIds[i],
+              stakingAccount: candidatesStakingAccounts[i],
+              rewardAccount: account,
+              stake: councilCandidateStake,
+            },
+          ]
+        })
       )
     )
-    await api.prepareAccountsForFeeExpenses(candidatesMemberAccounts, applyForCouncilTxs)
-    await api.sendExtrinsicsAndGetResults(applyForCouncilTxs, candidatesMemberAccounts)
+    await new FixtureRunner(announceCandidacyFixture).run()
 
     // Voting stage
     await this.api.untilCouncilStage('Voting')
 
+    const votersStakingAccounts = (await this.api.createKeyPairs(numberOfVoters)).map(({ key }) => key.address)
+    const optOutVoters = (await this.api.createKeyPairs(numberOfVoters)).map(({ key }) => key.address)
```

Comment:
I think there should be a separate flow testing the opt-out mechanics, instead of this being part of the `ElectCouncilFixture`. Or at least the opt-out testing part should be skipped by default, as the fixture is meant to be a minimal, reusable unit.